en/middleware/micrortps.md

```diff
@@ -66,21 +66,21 @@ The architecture for integrating ROS applications with PX4 is shown below.
 
 Note the use of [ros1_bridge](https://github.com/ros2/ros1_bridge), which bridges messages between ROS2 and ROS.
 This is needed because the first version of ROS does not support RTPS.
- 
+
 
 ## Code generation
 
+> **Note** [Fast RTPS must be installed](../setup/fast-rtps-installation.md) in order to generate the required code!
```

Comment:
This is true, but what isn't clear is that you don't need to do this "as an extra step" - it is automatically done in macOS installer, our ubuntu installation scripts and Cygwin (but not necessarily the other windows installation methods). 

Maybe: 

> **Note** [Fast RTPS must be installed](../setup/fast-rtps-installation.md) in order to generate the required code! 
> This is installed *by default* if you use the normal installers/scripts for [macOS](../setup/dev_env_mac.md), [Windows Cygwin](../setup/dev_env_windows_cygwin.md) or [Ubuntu](../setup/dev_env_linux_ubuntu.md).

Reply:
Yes that works!